src/sql/workbench/contrib/executionPlan/browser/topOperationsTab.ts

```diff
@@ -0,0 +1,406 @@
+		topOperationContainer.appendChild(tableContainer);
+		this._topOperationsContainers.push(topOperationContainer);
+
+		const rowNumberColumn = new RowNumberColumn({ numberOfRows: dataMap.length });
+		columns.unshift(rowNumberColumn.getColumnDefinition());
+
+		let copyHandler = new CopyKeybind<any>();
+		copyHandler.onCopy(e => {
+
+			const selectedDataRange = selectionModel.getSelectedRanges()[0];
+			let csvString = '';
+			if (selectedDataRange) {
+				const data = [];
+
+				for (let rowIndex = selectedDataRange.fromRow; rowIndex <= selectedDataRange.toRow; rowIndex++) {
+					const dataRow = table.getData().getItem(rowIndex);
+					const row = [];
+					for (let colIndex = selectedDataRange.fromCell; colIndex <= selectedDataRange.toCell; colIndex++) {
+						const dataItem = dataRow[table.columns[colIndex].field];
+						if (dataItem) {
+							row.push(dataItem.displayText ?? dataItem.text);
+						} else {
+							row.push(' ');
+						}
+					}
+					data.push(row);
+				}
+				csvString = data.map(row =>
+					row.map(x => `${x}`).join('\t')
+				).join('\n');
+
+				const columns = [];
+
+				for (let colIndex = selectedDataRange.fromCell; colIndex <= selectedDataRange.toCell; colIndex++) {
+					columns.push(table.columns[colIndex].name);
+				}
+
+			}
+
+			this._instantiationService.createInstance(CopyTableData).run({
+				selectedText: csvString
+			});
+		});
+
+		const selectionModel = new CellSelectionModel<any>();
+
+		const table = new Table<any>(tableContainer, {
+			columns: columns,
+			sorter: (args) => {
+				const column = args.sortCol.field;
+				dataMap.sort((a, b) => {
+					let result = -1;
+
+					if (!a[column]) {
+						result = 1;
+					} else {
+						if (!b[column]) {
+							result = -1;
+						} else {
+							const dataType = a[column].dataType;
+
+							const aText = a[column].displayText ?? a[column].text;
+							const bText = b[column].displayText ?? b[column].text;
+							if (aText === bText) {
+								result = 0;
+							} else {
+								switch (dataType) {
+									case sqlExtHostType.executionPlan.ExecutionPlanGraphElementPropertyDataType.String:
+									case sqlExtHostType.executionPlan.ExecutionPlanGraphElementPropertyDataType.Boolean:
+										result = aText.localeCompare(bText);
+										break;
+									case sqlExtHostType.executionPlan.ExecutionPlanGraphElementPropertyDataType.Number:
+										result = parseFloat(aText) - parseFloat(bText);
+										break;
+								}
+							}
+						}
+					}
+					return args.sortAsc ? result : -result;
+				});
+				table.setData(dataMap);
+			}
+		}, {
+			rowHeight: RESULTS_GRID_DEFAULTS.rowHeight,
+			forceFitColumns: false,
+			defaultColumnWidth: 120,
+			showRowNumber: true
+		});
+		table.setSelectionModel(selectionModel);
+		table.setData(dataMap);
+
+		table.registerPlugin(copyHandler);
+
+		table.setTableTitle(localize('topOperationsTableTitle', "Top Operations"));
+		table.onClick(e => {
```

Comment:
All handlers like this (and any other disposables) should be registered - which means this should be made into a disposable (and then your dispose method on the panel can actually do something)